further_link/util/user_config.py

```diff
@@ -89,6 +91,21 @@ def get_working_directory(user=None):
     return os.path.join(get_home_directory(user), DEFAULT_DIR_NAME)
 
 
+def get_miniscreen_projects_directory(name, user, username):
+    if user is None or not user_exists(user):
+        user = default_user()
+    if username is None:
+        username = "unknown"
+
+    base_dir = os.path.join(get_home_directory(user), DEFAULT_PROJECTS_DIR)
+
+    from_env = os.environ.get(FURTHER_LINK_MINISCREEN_PROJECTS_DIR)
+    if from_env is not None:
+        base_dir = from_env
+
+    return os.path.join(base_dir, username.split("@")[0], name)
```

Comment:
the @ symbol is a bit surprising here without the context. a comment wouldn't hurt